documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -150,7 +151,7 @@ Caution: if analyzer definitions have to be updated, the index will be closed au
 |`CREATE`|**The default**: existing indexes will not be altered, missing indexes will be created along with their mappings and analyzer definitions.
 |`RECREATE`|Indexes will be deleted if existing and then created along with their mappings and analyzer definitions.
 This will delete all content from the indexes!
-|`RECREATE_DELETE`|Similarly to `RECREATE` but will also delete the index at shutdown. Commonly used for tests.
+|`RECREATE_DELETE`|Similarly to `RECREATE` but will also delete the indexes at shutdown. Commonly used for tests.
```

Comment:
actually I think your change would be misleading: the option is per-index specific, you can choose a different property value for each index.